hw/dv/sv/dv_base_reg/dv_base_reg.core

```diff
@@ -17,6 +17,7 @@ filesets:
       - dv_base_mem.sv: {is_include_file: true}
       - dv_base_reg_block.sv: {is_include_file: true}
       - dv_base_reg_map.sv: {is_include_file: true}
+      - dv_base_lockable_field_cov.sv: {is_include_file: true}
```

Comment:
sorry am I blind? haha did you include this file in the PR?

Reply:
you're right. I forgot to add this file. Thanks